### ISSUE TITLE
evaluation doesnt function

### ISSUE THREAD

**Issue by SalocinB** [ttengwang/dense-video-captioning-pytorch#4]
Hi, I have a question about the evaluation. 

I tried the testing, and I didnt get errors, but the result I get is every time:

'METEOR', [0.0, 0.0, 0.0, 0.0]), ('Recall', [0.0, 0.0, 0.0, 0.0]), ('Precision', [0.0, 0.0, 0.0, 0.0]), ('tiou', [0.3, 0.5, 0.7, 0.9])

Can you help me to understand, what I am doing wrong?

 Thank you in advance.

**Comment by patrickESM** [comment]
Hello @ttengwang ,


in addition to SalocinB's comment i'd like to add:

this result occurs when calling 

`python eval.py --eval_folder tsrm_cmg_hrnn_RL_enlarged_trainset 
 --load_tap_json data/generated_proposals/tsn_dbg_esgn_testset_num5044.json
 --eval_caption_file data/captiondata/fake_test_anno.json`

the ActivityNet features where downloaded using the script:
`data/download_tsn_features.sh`

and the annotation files and pre-generated proposal were downloaded from Google Drive.

We also downloaded the model provided under the Pre-trained model section

**Comment by ttengwang** [comment]
Hi, thanks for your attention.

The zero results are normal because annotations of the test set are held by ActivityNet organizers for ActivityNet Challenge. If you want to evaluate on the test set, you may go to [http://activity-net.org/challenges/2021/evaluation.html](http://activity-net.org/challenges/2021/evaluation.html).

**Comment by ttengwang** [comment]
I just found that the evaluation server is unavailable after the deadline of this year's Challenge. :( you may contact the organizers for help.

**Comment by SalocinB** [comment]
Okay thank you for your promt reply!
